charts/internal/gvisor/values.yaml

```diff
@@ -1,4 +1 @@
-config:
-  kubernetesVersion: 1.16.1
```

Comment:
You could also update https://github.com/gardener/gardener-extension-runtime-gvisor/blob/d72a1e54ae1d90ecf94b92c21c0c727778f0afce/hack/api-reference/config.json#L20 (similar to the update you did for the other extensions).